Library/Homebrew/test/utils/shell_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "utils/shell"
+
+describe Utils::Shell do
+  describe "::path_to_shell" do
```

Comment:
On the contrary, I have never seen `.method`. RubyDoc uses `::` to describe class methods, e.g. https://ruby-doc.org/core-2.0.0/File.html.